lib/connector/rjmg16x.yaml

```diff
@@ -0,0 +1,17 @@
+name: RJMG16x
+datasheet: http://www.farnell.com/datasheets/71713.pdf
+description: Shielded 8P8C Modular Connector with Magnetics and LEDs
+ordercodes: [["Farnell", "1357436"]]
+footprint: agg:RJMG16X
+pins:
+- - - [LED1_C, 7, passive]
+    - [LED1_A, 8, passive]
+  - - [LED2_C, 9, passive]
+    - [LED2_A, 10, passive]
+  - - [SHIELD, S, passive]
+- - - [Tct, 1, passive]
+    - [Tx+, 2, passive]
+    - [Tx-, 3, passive]
+    - [Rct, 6, passive]
+    - [Rx+, 4, passive]
+    - [Rx-, 5, passive]
```

Comment:
I completely forgot I flipped this symbol in my design, so I'll flip those around. This is what it currently looks like, flipped:

![image](https://cloud.githubusercontent.com/assets/45057/26470601/40213d64-4196-11e7-9d7a-485c96318356.png)

From what I can tell, this kind of schematic layout is fairly standard, e.g. from the LAN8720 datasheet:

![image](https://cloud.githubusercontent.com/assets/45057/26471031/dcb37466-4197-11e7-9ac0-ca04213d717e.png)

I might move the LAN output pins to the top of the LAN8270A symbol so that it's easier to route those pull-ups more logically from the top.